styles/wiki.nixos.org/catppuccin.user.css

```diff
@@ -15,17 +15,31 @@
 @var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
 ==/UserStyle== */
 
-@-moz-document domain('nixos.wiki') {
-  @import url("https://python.catppuccin.com/pygments/catppuccin-variables.important.css");
-
+@-moz-document domain('wiki.nixos.org') {
+  /* This section applies the light/dark themes automatically based on the user's system theme.
+     Use the below example CSS to start if the website you are theming doesn't have a light/dark mode toggle.
+     For an actual example of this, see https://github.com/catppuccin/userstyles/blob/39288b593278e5efa48007d5fc001292cf910672/styles/brave-search/catppuccin.user.css#L16-L29.
+  */
   @media (prefers-color-scheme: light) {
     :root {
       #catppuccin(@lightFlavor, @accentColor);
+      --home-panel-heading-background: @mantle;
+      --home-panel-border-color: @crust;
+      --table-border-color: @crust;
+      --table-header-background: @surface1;
+      --table-even-background: @surface2;
+      --background-color-base: @base;
```

Comment:
if these need to be in `:root` you can put them in the `#catppuccin()` mixin without any additional selectors

Reply:
Done